config/routes.rb

```diff
@@ -0,0 +1,30 @@
+Rails.application.routes.draw do
+  root 'users#index'
```

Comment:
It appears that you're using the users controller/view as the front of the website.  A more typical approach to this is to create a welcome controller that displays the front page.